client/web/src/featureFlags/featureFlags.ts

```diff
@@ -18,6 +18,7 @@ export type FeatureFlagName =
     | 'search-results-keyboard-navigation'
     | 'enable-streaming-git-blame'
     | 'plg-enable-add-codehost-widget'
+    | 'hide-observability-only-visible-to-sourcegraph-operator'
```

Comment:
```suggestion
    | 'sourcegraph-operator-only-observability'
```

Then all our operator-related FFs can have the same prefix